openshift/setup-teardown/params/CUSTOM_SETTINGS/EAGLE_OS4/admin/tools/admin-1-angular-builder-bc.params

```diff
@@ -0,0 +1,3 @@
+NAME=eagle-admin-angular-builder
+GIT_REPO_URL=https://github.com/bcgov/eagle-admin
+GIT_REPO_BRANCH=develop
```

Comment:
Do we care about this "no new line at end of file" message?

Reply:
Not really, just means there is no character return at the end of this file.

Reply:
okok, thanks